docs/user/analytic_queries.rst

```diff
@@ -0,0 +1,245 @@
+Analytic Querying
+=================
+
+GeoMesa provides advanced query capabilities through GeoTools query hints. You can use these hints to control
+various aspects of query processing, or to trigger distributed analytic processing.
+
+.. _analytic_queries:
+
+Setting Query Hints
+-------------------
+
+Query hints can be set in two ways - programmatically or through GeoServer requests.
+
+Programmatic Hints
+^^^^^^^^^^^^^^^^^^
+
+To set a hint directly in a query:
+
+.. code-block:: java
+
+    import org.geotools.data.Query;
+
+    Query query = new Query("typeName");
+    query.getHints().put(key, value);
+
+Note that query hint values must match the class type of the query hint. See below for available hints and their types.
+
+GeoServer Hints
+^^^^^^^^^^^^^^^
+
+To set a hint through GeoServer, modify your query URL to use the ``viewparams`` request parameter:
+
+.. code-block:: none
+
+    ...&viewparams=key1:value1;key2:value2;
+
+Hint values will be converted into the appropriate types. See below for available hints and their accepted values.
+
+Loose Bounding Box
+------------------
+
+By default, GeoMesa uses a less precise filter for primary bounding box queries. This is faster, and in most cases
+will not change the results returned. However, certain use-cases require an exact result. This can be set
+at the data store configuration level, or overridden per query.
+
+===================== =========== =====================
+Key                   Type        GeoServer Conversion
+===================== =========== =====================
+QueryHints.LOOSE_BBOX ``Boolean`` ``true`` or ``false``
```

Comment:
Maybe it would be a good idea to put the exact argument for the ``viewparams`` parameter in the GeoServer Conversion column (e.g. ``LOOSE_BBOX:true``).